### PR TITLE
Fixing Android binding and adding DEMO project

### PR DESCRIPTION
**Pull request by icalderond**
Changes for this PR
- Update to Android: Meeting SDK Version: 5.12.2.9109 
- Adding demo for Android #9 
- Adding direct reference from binding to demo project #13

### CODE REVIEW

**Comment by icalderond** [comment]
@mdima @AdamDiament could  you review my PR to be integrated please

**Comment by mdima** [comment]
Hi @icalderond, 
only @AdamDiament can review your PR.

File: README.md
Comment (AdamDiament) [inline README.md:12]: @icalderond Is this removed because you added exoplayer slim? If so that is great! Do we still need to include the dummy resource files in the Xamarin.Android project to build?

File: samples/ZoomSDKSampleApp/ZoomSDKSampleApp.Android/ZoomSDKSampleApp.Android.csproj
Comment (AdamDiament) [inline samples/ZoomSDKSampleApp/ZoomSDKSampleApp.Android/ZoomSDKSampleApp.Android.csproj:97]: @icalderond Does the sample app need to reference the ZoomCommonLib directly itself? MobileRTC_Droid already references ZoomCommonLib itself so it should be fine without? Or am I mistaken?

**Review by AdamDiament — CHANGES_REQUESTED** [review]
Thanks very much for this.  In general it looks like great progress. I was able to run the sample app and start a zoom call in default UI mode. I also like how you versioned the name of the aar file, that's smart. However, when I tried to reference the created Android nuget package in my own custom ui integration, I get this error and cannot perform the nuget update to v5.12.2.9109

image

When I load the solution, I get the same error in the output for the sample app, but it doesn't stop me building or running the sample app. I guess nuget is a bit less forgiving. But until this is resolved we can't publish a new package so I can't merge without a solution to this.

To test this, the nuget package is located in the build output at Xamarin.ZoomSDKBinding\src\Droid\MobileRTC_Droid\bin\Release - you will need to set up a local nuget folder to be able to consume the package from your local machine *

Apart from this, I left a few more minor comments for your review. Particularly important is the one about removing the instructions to clean the aar file of bad placeholders.

Finally, I had to make an edit to the debug configuration file so that the sample app was set to deploy. I committed but I can't push because I don't have write access to your fork. Can you either do that change yourself or grant me access and let me know?

If you can address these issues, we should be good to merge.

Thanks again,

Adam

* (As an aside, I did manage to push a change straight on the github website to your fork to update the nuget package version in the mobilertc csproj by the way - so you can pull my change. I wonder why I can push directly to your fork on github.com but not push changes via the command line?)

File: README.md
Comment (icalderond) [reply README.md:221]: I tested the before versions an it was required use this tools to clean the .aar to be used but now is not required. I dont know if you needed to do this cleaning. If this is required for you test then we need keep in README file.

File: README.md
Comment (icalderond) [reply README.md:12]: It is correct. I added exoplayer slim and it not was required include dummy resources.

File: samples/ZoomSDKSampleApp/ZoomSDKSampleApp.Android/ZoomSDKSampleApp.Android.csproj
Comment (icalderond) [reply samples/ZoomSDKSampleApp/ZoomSDKSampleApp.Android/ZoomSDKSampleApp.Android.csproj:97]: It seems is not possible keep only RTC .aar file because when ia removed the another one (ZoomCommonLib.aar) the project show this error.

image

Comment (AdamDiament) [reply samples/ZoomSDKSampleApp/ZoomSDKSampleApp.Android/ZoomSDKSampleApp.Android.csproj:97]: Ok cool

**Comment by icalderond** [comment]
Thanks @AdamDiament for feedback. I sent invite to access to the fork. Please feel free to do changes to integrate this PR because i need confirm that this work correctly. Dont worry if you need undo some changes.

File: README.md
Comment (AdamDiament) [reply README.md:221]: Ok makes sense. It's possible that zoom finally cleaned up their resources file. But I have seen in the past that with new versions they reintroduce the same old bugs, and I don't want to lose the instructions on how to fix. Can you add them back in, but put a note in to say it only _may_ be necessary, rather than is required please?

**Comment by AdamDiament** [comment]
> Thanks @AdamDiament for feedback. I sent invite to access to the fork. Please feel free to do changes to integrate this PR because i need confirm that this work correctly. Dont worry if you need undo some changes.

Thanks. I pushed my change to set the sample app to deploy to your fork. I've replied to a couple of comments. The main outstanding things now is updating the readme to restore the instructions on cleaning the aar, and the version conflict for AndroidX.Core stopping the nuget package being consumed. I can have a look at the nuget issue, myself but I don't know when I will get to it, it might be at least a couple of weeks before I have time. So if you are able to find a solution yourself, great.

File: README.md
Comment (icalderond) [reply README.md]: Done. Please could you check my last commit with README file changes please

**Comment by icalderond** [comment]
> > Thanks @AdamDiament for feedback. I sent invite to access to the fork. Please feel free to do changes to integrate this PR because i need confirm that this work correctly. Dont worry if you need undo some changes.
> 
> Thanks. I pushed my change to set the sample app to deploy to your fork. I've replied to a couple of comments. The main outstanding things now are updating the readme to restore the instructions on cleaning the aar, and the version conflict for AndroidX.Core stopping the nuget package being consumed. I can have a look at the nuget issue myself but I don't know when I will get to it, it might be at least a couple of weeks before I have time. So if you are able to find a solution yourself, great. Then I could publish the new v. of the nuget.

Done. It was resolved conflict with AndroidX.Core and i return instructions from README file.

Please let me know if you could review that it work well.

**Comment by AdamDiament** [comment]
@icalderond works! I added a warning to the readme about the new androidx.core requirement as it's out of bounds of XF support. I hope we can get MAUI in soon as I'm getting nervous zoom will get too far ahead of us with their dependencies soon 😃 

Thank you very much for your work on this, it's a real improvement, especially with the polyfill files gone now.

I have published the latest nuget package also for implementation into your projects.

**Pull request merged**
